src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -4208,22 +4208,17 @@ public File[] parse(File outputDirectory, String[] classPath, Class cls) throws
         ClassProperties allProperties = Loader.loadProperties(cls, properties, true);
         ClassProperties clsProperties = Loader.loadProperties(cls, properties, false);
 
-        List<String> excludes = new ArrayList<>();
-        excludes.addAll(clsProperties.get("platform.exclude"));
-        excludes.addAll(allProperties.get("platform.exclude"));
+        Set<String> excludes = new HashSet<>(allProperties.get("platform.exclude"));
```

Comment:
`allProperties` already contains what is in `clsProperties`. And use an `HashSet` instead of an ArrayList since `excludes` is only used for `.contains(...)`